src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java

```diff
@@ -96,5 +100,14 @@ public boolean isAtAngle(Rotation2d targetHeight) {
         return Math.abs(targetHeight.getRadians() - getAngleInRadians()) <= ElbowConstants.TOLERANCE;
     }
 
+    public Pose3d getPose3D (){
+        return new Pose3d(
+                new Translation3d(
+                        0,0.1,0.6
+                ),
```

Comment:
move to a constant of PIVOT_TO_ROBOT_TRANSLATION

Reply:
its elbow and wrist btw